password.py

```diff
@@ -2,7 +2,7 @@
 from random import choice
 
 
-def generate_password(length: int, symbols: bool = False, numbers: int = 12) -> str:
+def generate_password(length: int, symbols: bool = False, numbers: bool = False) -> str:
```

Comment:
```suggestion
def generate_password(length: int, symbols: bool = False, numbers: int = 12) -> str:
```
Just to resolve merge conflict